src/documents/migrations/1045_alter_customfieldinstance_value_monetary.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.10 on 2024-02-22 03:52
+
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("documents", "1044_workflow_workflowaction_workflowtrigger_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Will this correctly change existing decimals to strings?

Reply:
Yea it worked fine testing with SQLite but youre right I should make a point to test with e.g. Postgres, there were some quirks with that recently.

before:
<img width="536" alt="Screenshot 2024-02-26 at 4 52 15 PM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/2cef97e5-dc29-481a-aa59-b02646a18a8d">
<img width="553" alt="Screenshot 2024-02-26 at 4 52 21 PM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/9118318f-c43a-48dd-8a98-bf2c78f4ec89">

after:
<img width="525" alt="Screenshot 2024-02-26 at 4 53 37 PM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/c5f766f5-fcc0-4444-8642-eb45a8ac4f06">
<img width="559" alt="Screenshot 2024-02-26 at 4 53 47 PM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/e6c2c64c-6725-403c-bec1-cf2ae1f3c026">

Reply:
Ok yea, tested with Postgres and was OK too. Let me know if you notice anything tho!

<img width="481" alt="Screenshot 2024-02-26 at 5 04 13 PM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/0a99200b-5fb5-4b99-8c43-7e3bcdb055ef">
<img width="474" alt="Screenshot 2024-02-26 at 5 05 45 PM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/3abec715-175d-483b-9c5d-5dd5b2077ee5">